_test/test_multifit/test_multifit_horace_dnd_1.m

```diff
@@ -0,0 +1,182 @@
+            mss = mss.set_bfun(@sqw_bcc_hfm, {[5,5,1.2,10,0],[5,5,1.4,15,0]}); % set background function(s)
+            mss = mss.set_bfree([1,1,1,1,1]);    % set which parameters are floating
+            mss = mss.set_bbind({1,[1,-1],1},{2,[2,-1],1});
+            
+            % Simulate at the initial parameter values
+            wsim_1 = mss.simulate();
+            
+            % And now fit
+            [wfit_1,fitpar_1] = mss.fit();
+            
+            % Test against saved or store to save later; ingnore string
+            % changes - these are filepaths
+            tol = [1e-10,1e-8];
+            assertEqualToTolWithSave (this, wsim_1, 'tol', tol, 'ignore_str', 1)
+            assertEqualToTolWithSave (this, wfit_1, 'tol', tol, 'ignore_str', 1)
+            assertEqualToTolWithSave (this, fitpar_1, 'tol', tol, 'ignore_str', 1)
+        end
+        
+        % ------------------------------------------------------------------------------------------------
+        function this = test_fit_two_datasets_ave(this)
+            % Example of simultaneously fitting more than one sqw object
+            % Average over pixels in a bin
+            
+            mss = multifit_sqw_sqw(this.win);
+            mss = mss.set_fun(@sqw_bcc_hfm,  [5,5,0,10,0]);  % set foreground function(s)
+            mss = mss.set_free([1,1,0,0,0]); % set which parameters are floating
+            mss = mss.set_bfun(@sqw_bcc_hfm, {[5,5,1.2,10,0],[5,5,1.4,15,0]}); % set background function(s)
+            mss = mss.set_bfree([1,1,1,1,1]);    % set which parameters are floating
+            mss = mss.set_bbind({1,[1,-1],1},{2,[2,-1],1});
+            mss.average = true;
+            
+            % Simulate at the initial parameter values
+            wsim_2 = mss.simulate();
+            
+            % And now fit
+            [wfit_2,fitpar_2] = mss.fit();
+            
+            % Test against saved or store to save later; ingnore string
+            % changes - these are filepaths
+            tol = [1e-10,1e-8];
+            assertEqualToTolWithSave (this, wsim_2, 'tol', tol, 'ignore_str', 1)
+            assertEqualToTolWithSave (this, wfit_2, 'tol', tol, 'ignore_str', 1)
+            assertEqualToTolWithSave (this, fitpar_2, 'tol', tol, 'ignore_str', 1)
+        end
+        
+        % ------------------------------------------------------------------------------------------------
+        function this=DISABLED_test_fit_array_of_datasets(this)
+            % Example of fitting single datasets, and independently fitting an array of datasets
+            % This exercises fit_sqw for single datasets and for an array of datasets
+
+            % *** NEEDS FIT_SQW TO BE IMPLEMENTED
+
+            % Check fit_sqw correctly fits array of input
+%             [wfit_single1,fitpar_single1]=fit_sqw(this.w1data, @sqw_bcc_hfm, [5,5,1,10,0], [0,1,1,1,1], @linear_bkgd, [0,0]);
```

Comment:
why are these disabled?

Reply:
@nickbattam-tessella Originally these tests were disabled as the loader had not been prepared. Now, having replaced fit_sqw (obsolete) with multifit_sqw, they fail as func_eval is not ported to d1d. Simple port would accept the conversion d1d->sqw->eval->d1d, but do we want to do that at this point?

Reply:
Would prefer disabling by skipTest